whats-new-panel.yaml

```diff
@@ -136,3 +136,43 @@
   targeting: firefoxVersion >= 80 && localeLanguageCode == "fr"
   trigger:
     id: whatsNewPanelOpened
+- id: WHATS_NEW_PIONEER_81
+  template: whatsnew_panel_message
+  order: 1
+  content:
+    bucket_id: WHATS_NEW_PIONEER_81
+    published_date: 1600732800000
+    title: Put your data to work for a better internet
+    icon_url: ""
+    icon_alt: ""
+    body: Contribute your data to Mozilla’s Pioneer program to help researchers understand pressing technology issues like misinformation, data privacy, and ethical AI.
+    cta_url: "pioneer"
+    cta_type: OPEN_ABOUT_PAGE
+    link_text: Join Pioneer
+  targeting: firefoxVersion >= 81 && localeLanguageCode == "en"
+  trigger:
+    id: whatsNewPanelOpened
+- id: WHATS_NEW_BADGE_81
+  content:
+    action:
+      id: show-whatsnew-button
+    badgeDescription:
+      string_id: cfr-badge-reader-label-newfeature
+    bucket_id: WHATS_NEW_BADGE_81
+    delay: 300000
+    target: whats-new-menu-button
+  frequency:
+    lifetime: 100
+  priority: 5
+  # Don't show the WNPanel notification if the user still has the FxA notification active
+  # Ok to show after 28 days, if the user interacted with the notification or sync is turned off
+  targeting:
+    firefoxVersion == 81 && (usesFirefoxSync || hasAccessedFxAPanel || currentDate|date
+    - 2419200000 >= profileAgeCreated) && (!messageImpressions['WHATS_NEW_BADGE_81']
+    || (messageImpressions['WHATS_NEW_BADGE_81']|length >= 1 && currentDate|date -
+    messageImpressions['WHATS_NEW_BADGE_81'][0] <= 345600000)) &&
+    "browser.messaging-system.whatsNewPanel.enabled"|preferenceValue &&
+    localeLanguageCode == "en"
```

Comment:
Just to confirm, this would narrow down the audience to be "en" only for this badge, is it because we only have one new panel item for "en'?

Reply:
yes because the new message will also be `en` only.